core/modules/admin_bar/admin_bar.inc

```diff
@@ -704,19 +704,28 @@ function admin_bar_links_alert() {
 function admin_bar_theme_settings() {
   $config = config('admin_bar.settings');
   $form['#config'] = 'admin_bar.settings';
+
   $form['margin_top'] = array(
     '#type' => 'checkbox',
     '#title' => t('Adjust top margin'),
     '#default_value' => $config->get('margin_top'),
     '#description' => t('Shifts the entire site content down to make room for the administration bar. If disabled, absolute- or fixed-positioned page elements may be covered by the administration bar.'),
   );
+
   $form['position_fixed'] = array(
     '#type' => 'checkbox',
     '#title' => t('Keep bar at top of page'),
     '#default_value' => $config->get('position_fixed'),
     '#description' => t('Displays the administration bar always at the top of the browser viewport, even when scrolling the page.'),
   );
 
+  $form['back_to_site'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Add Back To Site Link'),
```

Comment:
should be lowercased, as @BWPanda said in his [review](https://github.com/backdrop/backdrop-issues/issues/2709#issuecomment-390500108)